dm_cli/import_entity.py

```diff
@@ -96,16 +103,14 @@ def import_folder_entity(
     destination_path = Path(destination)
     data_source = destination_path.parts[0]
 
-    try:  # Check if target already exists on remote. Then delete or raise exception
-        dmss_api.document_get(f"dmss://{destination}/{source_path.name}")
+    # Check if target already exists on remote. Then delete or raise exception
+    target = f"{destination}/{source_path.name}"
+    exists = dmss_api.document_check(target)
```

Comment:
Don't think we should use this endpoint.

Reply:
Why not? :)